src/new_json.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import print_function
+
+import os
+import re
+import json
+from os.path import join, exists
+
+from utils import clean_input, is_empty, open_fr, ENCODING
+
+
+def short_names():
+    search_path = 'scripts'
+    shortnames = set()
+    if exists(search_path):
+        data_packages = [file_i for file_i in os.listdir(search_path) if file_i.endswith(".json")]
+        for script in data_packages:
+            script_name = '.'.join(script.split('.')[:-1])
+            script_name = script_name.replace('_', '-')
+            shortnames.add(script_name)
+        files = [file for file in os.listdir(search_path)
+                 if file[-3:] == ".py" and file[0] != "_" and
+                 ('#retriever' in
+                  ' '.join(open_fr(join(search_path, file), encoding=ENCODING).readlines()[:2]).lower())
+                 ]
+        for script in files:
+            script_name = '.'.join(script.split('.')[:-1])
+            script_name = script_name.replace('_', '-')
+            shortnames.add(script_name)
+    return list(shortnames)
+
+
+def get_replace_columns(dialect):
+    """Get the replace values for columns from the user."""
+    val = clean_input("replace_columns (separated by "
+                      "';', with comma-separated values) "
+                      "(press return to skip): ",
+                      split_char=';', ignore_empty=True)
+    if val == "" or val == []:
```

Comment:
`if val == "" or val == []:` I  have seen a lot of these. we can clean them after we have shipped the code and merged since you did not create those parts.